models/service.go

```diff
@@ -125,6 +126,29 @@ type (
 	}
 )
 
+// Reduced information for a service
+// To create links to another service
+// Used by Ambient, to indicate the waypoint proxies
```

Comment:
```suggestion
// ServiceInfo holds the service information needed to create links to another service. Used, for example, to link services to Ambient waypoint waypoint proxies
```

Maybe this type name should be more specific?  ServiceLinkInfo, ServiceReferenceInfo?

Reply:
Thanks for the suggestion, @jshaughn ! I have changed for ServiceReferenceInfo